factoring/interfaces.py

```diff
@@ -92,7 +92,7 @@ def factor(P, use_saved_embedding=True):
     sample_time = time.time()
 
     # get QPU sampler
-    sampler = DWaveSampler(solver=dict(qpu=True))
+    sampler = DWaveSampler()
```

Comment:
This actually requires [Ocean 3.0.0](https://github.com/dwavesystems/dwave-ocean-sdk/releases/tag/3.0.0)+, i.e. dwave-system 1.0+.